release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -364,6 +364,85 @@ This dashboard provides API server metrics, such as:
+* The `PrometheusOperatorNotReady` alert is added. The warning alert provides notification if the Prometheus Operator in your namespace is not ready.
+* The `PrometheusOperatorRejectedResources` alert is added. The warning alert provides notification if specific resources are rejected by the Prometheus Operator.
+* The `PrometheusOperatorSyncFailed` alert is added. The warning alert provides notification if the last controller in your namespace failed to reconcile specific objects.
+* The `PrometheusTargetLimitHit` alert is added. The warning alert provides notification if Prometheus has dropped targets because some scrape configurations have exceeded the limit of the targets.
+* The `ThanosSidecarPrometheusDown` alert is added. The critical alert provides notification that the Thanos sidecar cannot connect to Prometheus.
+* The `ThanosSidecarUnhealthy` alert is added. The critical alert provides notification that the Thanos sidecar is unhealthy for a specified amount of time.
+* The `NodeClockNotSynchronising` alert is updated to prevent false positives in environments that use the chrony time service (chronyd).
+* The `NodeNetworkReceiveErrs` alert is updated to ensure that the alert does not fire when only a small number of errors are reported. The rule now uses the ratio of errors to total packets instead of the absolute number of errors.
+* The `NodeNetworkTransmitErrs` alert is updated to ensure that the alert does not fire when only a small number of errors are reported. The rule now uses the ratio of errors to total packets instead of the absolute number of errors.
+* The `etcdHighNumberOfFailedHTTPRequests` alerts with severities of *warning* and *critical* are removed. These alerts fired if a high percentage of HTTP requests failed on an etcd instance.
+
+[NOTE]
+=====
+Red Hat does not guarantee backward compatibility for metrics, recording rules, or alerting rules.
+=====
+====
+
+[id="ocp-4-7-monitoring-stack-dependency-version-updates"]
+==== Version updates to monitoring stack components and dependencies
+
+{product-title} 4.7 includes version updates to the following monitoring stack components and dependencies:
+
+* The Prometheus Operator is now on version 0.44.1.
+* Thanos is now on version 0.17.2.
+* `kube-prometheus` is now on release 0.7.
+
+[id="ocp-4-7-monitoring-prometheus-operator-alertmanagerconfig-crd-"]
+==== AlertmanagerConfig CRD in Prometheus Operator not supported
+
+Modifying Alertmanager configurations by using the `AlertmanagerConfig` custom resource definition (CRD) in the Prometheus Operator is not supported.
+
+For more information, see xref:../monitoring/configuring-the-monitoring-stack.html#support-considerations_configuring-the-monitoring-stack[Support considerations for monitoring].
+
+[id="ocp-4-7-monitoring-namespace-by-pod-and-pod-total-Grafana-dashboards"]
+==== Namespace By Pod and Pod Total Grafana dashboards
+
+The Namespace By Pod and Pod Total networking Grafana dashboards are now enabled.
```

Comment:
Should these dashboard names be marked up as a `*UI element*`?

Reply:
Yes, good catch!